src/apps/sommelier/sommelierApi.ts

```diff
@@ -0,0 +1,86 @@
+import { Address } from 'viem'
+import { NetworkId } from '../../types/networkId'
+import got from '../../utils/got'
+
+const NETWORK_ID_TO_SOMM_CHAIN: Record<NetworkId, string | null> = {
+  [NetworkId['celo-mainnet']]: null,
+  [NetworkId['ethereum-mainnet']]: 'ethereum',
+  [NetworkId['arbitrum-one']]: 'arbitrum',
+  [NetworkId['op-mainnet']]: 'optimism',
+  [NetworkId['polygon-pos-mainnet']]: null,
+  [NetworkId['base-mainnet']]: null,
+  [NetworkId['celo-alfajores']]: null,
+  [NetworkId['ethereum-sepolia']]: null,
+  [NetworkId['arbitrum-sepolia']]: null,
+  [NetworkId['op-sepolia']]: null,
+  [NetworkId['polygon-pos-amoy']]: null,
+  [NetworkId['base-sepolia']]: null,
+}
+
+export async function getSommStrategiesData(networkId: NetworkId) {
+  const { result } = await got
+    .get(`https://app.somm.finance/api/sommelier-api-all-strategies-data`)
+    .json<{
+      result: {
+        data: {
+          cellars: {
+            id: Address
+            shareValue: string
+            tvlTotal: number
+            chain: string
+          }[]
+        }
+      }
+    }>()
+
+  return result.data.cellars
+    .filter((cellar) => cellar.chain === NETWORK_ID_TO_SOMM_CHAIN[networkId])
+    .map((cellar) => {
+      const cellarAddress = cellar.id.split('-')[0] as Address // some cellars have a suffix to the address (e.g. `-arbitrum`), which we do not need
+      return {
+        ...cellar,
+        address: cellarAddress,
+        strategySlug: cellarAddressToSlug[cellarAddress.toLowerCase()], // used to generate the manageUrl
+      }
+    })
+}
+
+// Derived from the hard coded config in the Sommelier website source code
+// https://github.com/PeggyJV/sommelier-strangelove/blob/ca7bd6605bc868a1393d820f13b341ae5a5f1ead/src/utils/config.ts,
+// filtered by the vaults returned by
+// https://app.somm.finance/api/sommelier-api-all-strategies-data
+export const cellarAddressToSlug: Record<string, string> = {
```

Comment:
If they add a new vault we'll need to add it
```suggestion
export const cellarAddressToSlug: Record<string, string | undefined> = {
```